libcxx/utils/ci/run-buildbot

```diff
@@ -136,6 +136,7 @@ function generate-cmake-base() {
           -DLIBCXX_ENABLE_WERROR=YES \
           -DLIBCXXABI_ENABLE_WERROR=YES \
           -DLIBUNWIND_ENABLE_WERROR=YES \
+          -DCMAKE_INSTALL_MESSAGE=NEVER \
```

Comment:
Can you please add a comment that we can remove this once https://gitlab.kitware.com/cmake/cmake/-/issues/26085 has been resolved in CMake (by passing it only in the `install-cxx-test-suite-prefix` target)?